examples/WORKSPACE

```diff
@@ -1,24 +1,21 @@
 workspace(name = "examples")
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
-http_archive(
+git_repository(
     name = "io_bazel_rules_rust",
-    sha256 = "a62597502c1ee14fcc30da92fc31c69bc6b436f464939449dc7be88c76218a31",
-    strip_prefix = "rules_rust-6e5fa2c570ac2f17ac1df840d060fc8aab521a07",
-    urls = [
-        # Master branch as of 2020-06-06
-        "https://github.com/bazelbuild/rules_rust/archive/6e5fa2c570ac2f17ac1df840d060fc8aab521a07.tar.gz",
-    ],
+    commit = "fda9a1ce6482973adfda022cadbfa6b300e269c3",
+    remote = "https://github.com/bazelbuild/rules_rust.git",
+    shallow_since = "1593540027 +0200",
 )
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 http_archive(
     name = "bazel_skylib",
-    sha256 = "9a737999532daca978a158f94e77e9af6a6a169709c0cee274f0a4c3359519bd",
-    strip_prefix = "bazel-skylib-1.0.0",
-    url = "https://github.com/bazelbuild/bazel-skylib/archive/1.0.0.tar.gz",
+    sha256 = "eb5c57e4c12e68c0c20bc774bfbc60a568e800d025557bc4ea022c6479acc867",
+    strip_prefix = "bazel-skylib-0.6.0",
+    url = "https://github.com/bazelbuild/bazel-skylib/archive/0.6.0.tar.gz",
```

Comment:
Should this just be updated in the smoke-test? It was bumped in a PR a while back for the examples, and it probably should have been bumped in the smoke-test then too.

Reply:
I updated `smoke-test.sh` so now it should be using the updated version of `bazel_skylib` for tests and match the example output. Sorry about not catching this before. I blindly ran `smoke-test.sh` and just committed whatever was generated 😅